integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/spatial/AbstractSpatialWithinSearchPredicateIT.java

```diff
@@ -112,14 +112,17 @@ protected void initData() {
 		final IndexFieldAccessor<GeoPoint> geoPoint_2;
 		final IndexFieldAccessor<GeoPoint> geoPoint_with_longName;
 		final IndexFieldAccessor<GeoPoint> nonProjectableGeoPoint;
+		final IndexFieldAccessor<GeoPoint> unsortableGeoPoint;
 		final IndexFieldAccessor<String> string;
 
 		IndexAccessors(IndexSchemaElement root) {
-			geoPoint = root.field( "geoPoint" ).asGeoPoint().projectable( Projectable.YES ).createAccessor();
-			geoPoint_1 = root.field( "geoPoint_1" ).asGeoPoint().projectable( Projectable.YES ).createAccessor();
-			geoPoint_2 = root.field( "geoPoint_2" ).asGeoPoint().projectable( Projectable.YES ).createAccessor();
-			geoPoint_with_longName = root.field( "geoPoint_with_a_veeeeeeeeeeeeeeeeeeeeerrrrrrrrrrrrrrrrrryyyyyyyyyyyyyyyy_long_name" ).asGeoPoint().projectable( Projectable.YES ).createAccessor();
+			geoPoint = root.field( "geoPoint" ).asGeoPoint().sortable( Sortable.YES ).projectable( Projectable.YES ).createAccessor();
```

Comment:
Do we really need the fields to be projectable or sortable for the *predicate* tests? That looks odd.

Reply:
It's also the base class for DistanceSearchProjectionIT. And we project and sort on some of those fields.

Reply:
Mkay. I guess that spares us some copy-pasting.